app/src/main/java/com/example/waterme/ui/WaterMeApp.kt

```diff
@@ -80,7 +81,7 @@ fun PlantListContent(
     LazyColumn(
         contentPadding = PaddingValues(dimensionResource(id = R.dimen.padding_medium)),
```

Comment:
Systembars does't seem to be transparent. 

<img width="283" alt="image" src="https://github.com/google-developer-training/basic-android-kotlin-compose-training-waterme/assets/24260141/391bcd38-71b6-49a3-a201-61b708a3c348">

Reply:
Please provide additional details on your setup and how you are testing.

The navigation bar is transparent on all versions I tested. Pixel 3 - API 24, 26, and 33

<img width="1298" alt="Screenshot 2023-08-25 at 9 36 53 AM" src="https://github.com/google-developer-training/basic-android-kotlin-compose-training-waterme/assets/7107817/6f34fa10-cdd0-4fbd-89dc-7856cbc6f49b">

Reply:
did you test in landscape mode? 
I don't see any landscape mode screenshots.

Reply:
re: test in landscape - Yes, I did. All APIs tested (Pixel 3 - API 24, 26, and 33) were translucent so screenshots were not attached.

The feedback left only showed one emulator screenshot in portrait mode. Nothing was mentioned about landscape.

Are you having issues in landscape too?

Please provide additional details on your setup and how you are testing.

Reply:
Fixed.